From 480e1653395e4943070605dc81f333046cb88112 Mon Sep 17 00:00:00 2001
From: Joey Riches <josephriches@gmail.com>
Date: Tue, 6 Oct 2026 21:40:39 +0100
Subject: [PATCH 1/1] freedesktop: Remove old launchable tag in appstream
 metainfo

Since appstream >= 1.2.x having more than one lauchable tag will cause
appstream compose to fail to compose the metainfo.

Remove the stale .desktop launchable, the correct launchable
`<launchable
type="desktop-id">org.freedroid.freedroidRPG.desktop</launchable>`

already exists.
---
 pkgs/freedesktop/org.freedroid.freedroidRPG.appdata.xml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/pkgs/freedesktop/org.freedroid.freedroidRPG.appdata.xml b/pkgs/freedesktop/org.freedroid.freedroidRPG.appdata.xml
index 78c78c37f..e498e45a3 100644
--- a/pkgs/freedesktop/org.freedroid.freedroidRPG.appdata.xml
+++ b/pkgs/freedesktop/org.freedroid.freedroidRPG.appdata.xml
@@ -84,7 +84,6 @@
     <content_attribute id="money-purchasing">none</content_attribute>
     <content_attribute id="money-gambling">moderate</content_attribute>
   </content_rating>
-  <launchable type="desktop-id">freedroidrpg.desktop</launchable>
   <provides>
     <binary>freedroidRPG</binary>
   </provides>
-- 
2.56.0

